CONTRIBUTING.md

```diff
@@ -10,33 +10,36 @@ For general information about the project, see [our readme](https://github.com/z
 Most discussions happen on our [Discord Server](https://discord.zelda64.dev) where you are welcome to ask if you need help getting started, or if you have any questions regarding this project and other decompilation projects.
 
 
-Useful Links
-------------
+## Useful Links
 
-- [Discord](https://discord.zelda64.dev/) - Primary discussion platform.
-- [Google Sheets reservation board](https://docs.google.com/spreadsheets/d/1X83YCPRa532v-Zo0WgUsJ2kB1X9RxBta5_p9aWA8uro) - We use this to track decompilation progress, not GitHub Issues.
+- [Google Sheet](https://docs.google.com/spreadsheets/d/1X83YCPRa532v-Zo0WgUsJ2kB1X9RxBta5_p9aWA8uro) - We use this to track decompilation progress and file reservations, not GitHub Issues or Trello or similar.
```

Comment:
```suggestion
- [Google Sheet](https://docs.google.com/spreadsheets/d/1X83YCPRa532v-Zo0WgUsJ2kB1X9RxBta5_p9aWA8uro) - We use this to track decompilation progress and file reservations, instead of GitHub Issues or Trello or similar.
```
maybe?